src/platform/nrfconnect/wifi/NrfWiFiDriver.cpp

```diff
@@ -27,6 +28,9 @@
 using namespace ::chip;
 using namespace ::chip::DeviceLayer::Internal;
 using namespace ::chip::DeviceLayer::PersistedStorage;
+using namespace ::chip::app::Clusters::NetworkCommissioning;
+
+constexpr static auto WiFiCommissioningTimeoutMs = 10000;
```

Comment:
1. WiFiCommissioningTimeoutMs-> kWiFiCommissioningTimeoutMs
2. Can we move it to the NrfWifiDriver.h, as a class field?

Reply:
Hmm or maybe we should read it from a specific cluster? What do you think, @markaj-nordic ?

Reply:
The NrfWiFiDriver.h already has a `constexpr uint8_t kWiFiConnectNetworkTimeoutSeconds = 120` which is used to set the corresponding cluster read by the controller during the commissioning. I guess we can end up with only one const for both cases and convert seconds/ms if needed. Also note that this is a **connection** timeout not **commissioning** timeout. So the naming should be `kWiFiConnectionTimeoutMs`